Makefile

```diff
@@ -266,8 +270,9 @@ test\:coverage-func:
 
 .PHONY: test\:coverage-html
 test\:coverage-html:
-	@$(MAKE) test:coverage
+	@$(MAKE) test:coverage path=$(path)
 	go tool cover -html=$(COVERAGE_FILE)
+	rm -rf ./coverage
```

Comment:
todo: `make clean:coverage` instead?

Reply:
I do want the html output to have a visual.